scripts/run-benchs.mjs

```diff
@@ -0,0 +1,38 @@
+/**
+ * @fileoverview This script is used to run all the benchmarks in the project.
+ * @link https://github.com/RafaelGSS/bench-node
+ */
+import { spawnSync } from 'node:child_process';
+import { styleText } from 'node:util';
+import { globSync } from 'node:fs';
+
+// hacky way to use work around npm workspaces
+const args = process.argv.slice(2);
+const w =
+	args
+		.find((arg) => arg.startsWith('--w='))
+		?.replace('--w=', '')
+		.trim()
+		.replace('package/', '') ?? '';
```

Comment:
We control this, so not terribly important, but would [`node:utils.parseArgs`](https://nodejs.org/api/util.html#utilparseargsconfig) work better for finding this? it supports a long and a short flag name

Reply:
i had tried with this api but I cannot make it work